src/components/v2/Layout/Header/styles.ts

```diff
@@ -3,11 +3,48 @@ import { useTheme } from '@mui/material';
 
 export const useStyles = () => {
   const theme = useTheme();
-  return css`
-    background-color: ${theme.palette.background.default};
-    width: ${theme.shape.layoutOffset.width};
-    margin-left: ${theme.shape.layoutOffset.ml};
-    background-image: none;
-    box-shadow: none;
-  `;
+  return {
+    appBar: css`
+      background-image: none;
+      background-color: transparent;
+      box-shadow: none;
+    `,
+    toolbar: css`
+      justify-content: space-between;
+      display: flex;
+
+      /* @TODO Remove when the ConnectButton is refactored */
+      .connect-btn {
+        height: auto;
+        margin: ${theme.spacing(2)};
+        padding: ${theme.spacing(1.5)} ${theme.spacing(3)};
+        width: 10.3rem;
+      }
+      ${theme.breakpoints.down('md')} {
+        display: none;
+      }
+    `,
+    rightItemPaper: css`
+      margin: ${theme.spacing(2)};
+      justify-content: center;
+      align-items: center;
+      padding: ${theme.spacing(1)};
+      display: inline-flex;
+      width: 10.3rem;
+    `,
+    coinInfo: css`
+      display: flex;
+      flex-direction: row;
+      align-items: center;
+      width: 10.3rem;
```

Comment:
Same thing here about using pixels. Also, this should be a `min-width` instead of a fixed `width`, so that if a user have a large amount of coins their balance can still be displayed without it overflowing.